gulpfile.js

```diff
@@ -1,44 +1,63 @@
-const gulp = require('gulp')
-const gutil = require('gulp-util')
-const gclean = require('gulp-clean')
-const typescript = require('gulp-typescript')
-const sourcemaps = require('gulp-sourcemaps')
-const gmocha = require('gulp-mocha')
-const tsnode = require('ts-node/register')
-const tsProject = typescript.createProject('tsconfig.json')
+/**
+ * @license
+ * Copyright 2017-2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const gulp = require('gulp');
+const gclean = require('gulp-clean');
+const typescript = require('gulp-typescript');
+const sourcemaps = require('gulp-sourcemaps');
+const gmocha = require('gulp-mocha');
+const tsnode = require('ts-node/register');
+const tsProject = typescript.createProject('tsconfig.json');
 
 const OPTIONS = {
 	dirs: {
 		sources: './src',
-		build: './build'
-	}
-}
+		build: './build',
+	},
+};
 
-gulp.task('test', () => {
-	return gulp.src('tests/tests.ts')
-		.pipe(gmocha({
-			compilers: [
-				'ts:ts-node/register'
-			]
-		}))
-})
+function test() {
```

Comment:
These changes from `gulp.task('test')` to `function test()` are to support Gulp 4: https://www.liquidlight.co.uk/blog/how-do-i-update-to-gulp-4/